sale_coupon_multi_gift/models/sale_coupon_multi_gift.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleCouponGift(models.Model):
+    _name = "sale.coupon.multi.gift"
+    _description = "Coupon Multi Gift"
+
+    coupon_reward_id = fields.Many2one(comodel_name="sale.coupon.reward")
+    reward_product_quantity = fields.Integer(
```

Comment:
This shouldn't be the requirement, but the reward quantity. You should also not consider multiple quantities. The gift is rewarded or not, not X times rewarded.

Reply:
Changes done :+1: